packages/node-build-scripts/sass-compile.ts

```diff
@@ -0,0 +1,40 @@
+import fs from "fs-extra";
```

Comment:
see https://github.com/palantir/blueprint/pull/5215/files#diff-8c508100032d0fe222dc9679ae03f037d137a2323c18decd3775ccea41dddf2eR14

> the `dart-sass` CLI doesn't support custom functions or importers, but the JS API does, so delegate to node

Very frustrating, but basically the only way we can migrate and keep `sass-inline-svg` and the importer (which allows us to resolve tilde imports outside of webpack), we need to delegate to node